superset/dashboards/filters.py

```diff
@@ -255,3 +256,86 @@ def apply(self, query: Query, value: Any) -> Query:
         if value is False:
             return query.filter(and_(Dashboard.created_by_fk.is_(None)))
         return query
+
+class DashboardIsRecommended(BaseFilter):
+    name = _("Is recommended")
+    arg_name = "dashboard_is_recommended"
+    limit = 5
+
+    def _get_most_viewed_in_last_month(self, user_id):
+        """Logic here is similar to get_recent_activity in superset/daos/log.py
+        but is adjusted to use 30 day time frame and return most viewed"""
+        one_month_ago = datetime.today() - timedelta(days=30)
+        subqry = (
+            db.session.query(
+                Log.dashboard_id,
+                func.count(Log.id).label('views')
+            )
+            .group_by(Log.dashboard_id)
+            .filter(
+                and_(
+                    Log.action.in_(["dashboard"]),
+                    Log.user_id == user_id,
+                    Log.dttm > one_month_ago,
+                    Log.dashboard_id.isnot(None)
+                )
+            )
+            .order_by(func.count(Log.id).desc())
+            .subquery()
+        )
+        qry = (
+            db.session.query(Dashboard.id)
+            .outerjoin(subqry, Dashboard.id == subqry.c.dashboard_id)
+            .filter(
+                and_(
+                    Dashboard.dashboard_title is not None,
```

Comment:
double check if it will behave correctly with `is not` in sqlalchemy, or should it be `!=`